test/services/testVideos.js

```diff
@@ -1,13 +1,19 @@
 /* eslint-env mocha */
 require('dotenv').config()
+require('sinon-mongoose')
 var sinon = require('sinon')
 // var chai = require('chai')
 // var expect = chai.expect
 var videoService = require('../../services/videos')
+var Videocall = require('../../models/videocall')
 
 describe('Video Service', function () {
   describe('generateChatName', function () {
     it('should return a unique chat name', sinon.test(function () {
+      sinon.mock(Videocall)
+        .expects('findOne')
+        .chain('exec')
+        .yields(null, null)
```

Comment:
haha I was thinking more on the lines of
1. When we call findOne, the first time it returns that it found something with the same name
2. The second time findOne, it returns null, null
3. Pass in a callback to generateChatName that checks that a string is returned